tests/test_codecs.py

```diff
@@ -208,19 +208,6 @@ def test_context_from_readable_headers(self):
                 'hermes': 'LaBarbara Hermes',
             }
 
-    def test_baggage_without_trace_id(self):
```

Comment:
moved down (outside of class) to be able to get `tracer` fixture. Not sure if the same possible when subclassing `TestCodecs(unittest.TestCase)`.